### ISSUE TITLE
Не хочет работать

### ISSUE THREAD

**Issue by xoceunder** [karpeychik/Kinopub#24]
Не хочет работать
@karpeychik

**Comment by Krishnacore** [comment]
> Не хочет работать
> @karpeychik

@xoceunder Привет! Возможно я нашел причину, по которой может не работать. Там есть момент, когда он делает POST и там, в том числе, в запросе есть инфа про устройство, название. У меня в нем всякое непотребство вроде пробелов и кавычек. Я сменил название устройства и все завелось

**Comment by xoceunder** [comment]
@Krishnacore @karpeychik @ukarpeychik 

Не могли бы вы помочь мне реализовать функцию открытия отдельных категорий, чтобы раздел фильмов был быстрее?

**Comment by nzholdas** [comment]
У меня просто серый экран при включении. Хотя при первом запуске вышел код регистрации для кинопаб, но с тех пор только серый экран. В чём проблема?  У вас работает? @karpeychik @xoceunder @krishnacore

**Comment by Krishnacore** [comment]
> У меня просто серый экран при включении. Хотя при первом запуске вышел код регистрации для кинопаб, но с тех пор только серый экран. В чём проблема?  У вас работает?

Да, работает. Как я выше писал, попробуйте глянуть название устройства - там, похоже, не должно быть пробелов, кавычек и всего такого

**Comment by nzholdas** [comment]
@Krishnacore 
Спасибо что быстро ответили) хорошо узнал что хотя бы работает
Не знаю, может туплю сильно. Но у моего року вообще названия вроде как нет, в настройках просто имейл вписан. А в кинопабе написано название unknown устройство. Я вообще не уверен то ли я скачал. Вы скачали файл kinopub/kinopub/kinopub.zip? Или весь код скачали и запихали папку kinopub в zip? И так и так пробовал, но хз. Файлы подозрительно мало весят, 137 и 319 кб. Так и должно быть?

**Comment by Krishnacore** [comment]
> Спасибо что быстро ответили) хорошо узнал что хотя бы работает
> Не знаю, может туплю сильно. Но у моего року вообще названия вроде как нет, в настройках просто имейл вписан. А в кинопабе написано название unknown устройство. Я вообще не уверен то ли я скачал. Вы скачали файл kinopub/kinopub/kinopub.zip? Или весь код скачали и запихали папку kinopub в zip? И так и так пробовал, но хз. Файлы подозрительно мало весят, 137 и 319 кб. Так и должно быть?

Я в итоге это приложение чуть ковырял и под свои нужды правил, и собирал его заново, но тот zip должен работать. 

Хм, кажется, через настройки самого устройства имя не сменить. Я менял в приложении на телефоне

**Comment by nzholdas** [comment]
Уфф, заработало. И правда, через приложение только увидел имя устройства и сменил. Спасибо большое!
А у вас тоже поиска нет? И ещё зависает если зайти в закладки, в 4к и 3д не включаются. Не поделитесь своим зип? Был бы премного благодарен @nooorj в телеге

**Comment by klimovitsky** [comment]
1. Go to "https://my.roku.com/account" 
2. Login to your Roku account
3. At the bottom of the page, under the "My linked devices" find your device, and click on the edit icon (circle with pen). 
4. Rename the device to anything without spaces or other special characters (i.e. Roku). 
5. Restart your Roku device.
6. Remove the Application by clicking on the Delete button on the "Development Application Installer" page where you installed this Github channel/app. 
7. Install the channel/app again
8. Should work.

**Comment by nzholdas** [comment]
I got it working thanks to krishnacore but thanks anyway!

**Comment by xoceunder** [comment]
> 

Install the code but it doesn't show anything

**Comment by quackybun** [comment]
Спасибо за подсказку с пробелами в имени Року девайса.
Однако, результат "превзошел все ожидания" - показывает просто каталог каких-то фильмов, не существует поиска, настроек, или чего-то ещё. По информативности напоминает табло в торговом центре - есть фильмы, вот, 10-20 штук, можно посмотреть их безо всяких опций, просто вкл/выкл. Меню "закладки" вешает приложение.
